### DISCUSSION TITLE
What can I access from `context`?

### DISCUSSION THREAD

**Discussion by szalai1** [dagster-io/dagster#2951] [Q&A] [answered]
I couldn't find any documentation about `context`. I found code snippets accessing `run_id` and `resources. I'm wondering what else is there (and supported). 

Thanks

**Comment by alangenfeld** [comment, marked as answer]
https://docs.dagster.io/_apidocs/execution#execution-context

the available info is spread across `SolidExecutionContext` and its base class `AbstractComputeExecutionContext`

**Reply by helloworld** [reply]
More info here as well: https://docs.dagster.io/overview/solids-pipelines/solids#solid-context

**Reply by mateusz-krainski-revoize** [reply]
this link (https://docs.dagster.io/overview/solids-pipelines/solids#solid-context) returns a 404 now. If I'm reading it right, Solid pipelines were deprecated ([link](https://docs.dagster.io/guides/migrate/version-migration#legacy-api-removals)).

Just leaving a note for whoever may wander here in the future.
